SkillChat.Server/NETVersionComparison.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+
+namespace SkillChat.Server
+{
+    public class NETVersionComparison
+    {
+        /// <summary>
+        /// Получение наиболее подходящего Runtime .NET
+        /// </summary>
+        public static Runtime GetBestDotNETRuntime(string cfgVersion)
+        {
+            Runtime cfgRuntime = new Runtime() {Version = cfgVersion};
```

Comment:
Тут тоже логичнее использовать System.Version, чтобы сравнивать с целевой.
```suggestion
            var cfgRuntime = new Version(cfgVersion);
```